### PR TITLE
Support relative paths to manifest files

### PR DESCRIPTION
**Pull request by kmod**

### CODE REVIEW

File: pyperformance/_manifest.py [hunk 1]
```diff
@@ -275,8 +275,7 @@ def _iter_sections(lines):
 
 
 def _parse_manifest_file(filename):
-    relroot = os.path.dirname(filename)
-    filename = _utils.resolve_file(filename, relroot)
+    filename = os.path.abspath(filename)
```
Comment (ericsnowcurrently) [inline pyperformance/_manifest.py:278]: I'd say drop both lines here and add `filename = _utils.resolve_file(filename)` up in `load_manifest()`.

**Review by ericsnowcurrently — CHANGES_REQUESTED** [review]

**Comment by ericsnowcurrently** [comment]
Thanks for working on this!

**Comment by kmod** [comment]
Sure thing, how about this?

**Review by ericsnowcurrently — APPROVED** [review]
LGTM

That works for you?

**Comment by kmod** [comment]
yep! I tested it and it seems to work

**Comment by ericsnowcurrently** [comment]
Thanks, @kmod!

**Pull request merged**